manifest/989onan/NoKnockBack/info.json

```diff
@@ -0,0 +1,23 @@
+{
+	"name": "989onan.NoKnockBack",
+	"id": "989onan.NoKnockBack",
+	"description": "A Resonite mod that prevents knockback and sends an impulse of \"knockbackcanceled\" for knockback and \"gravitychangecanceled\" for gravity changes to the client's user root.",
+	"category": "Misc",
+	"sourceLocation": "https://github.com/989onan/NoKnockBack",
+	"versions": {
+		"1.0.0": {
```

Comment:
For the initial additions, add only the latest version for each mod. Adding all previous versions is not needed